mantidimaging/gui/windows/live_viewer/live_view_widget.py

```diff
@@ -35,7 +35,8 @@ def show_image(self, image: np.ndarray) -> None:
         Show the image in the image view.
         @param image: The image to show
         """
-        self.image.setImage(image)
+        self.image.use_brightness_percentiles(image, 5, 95)
+        self.image.setImage(image, levels=self.image.brightLevels)
```

Comment:
Can `MIMiniImageView.setImage()` do a check to see if the percentiles are being used, so that levels does not need to be passed in from outside. So this part would be 

```
self.image.use_brightness_percentiles(image, 5, 95)
self.image.setImage(image)
```